isisd/isis_cli.c

```diff
@@ -87,9 +89,11 @@ DEFPY_YANG(no_router_isis, no_router_isis_cmd,
 
 	nb_cli_enqueue_change(vty, ".", NB_OP_DESTROY, NULL);
 
-	return nb_cli_apply_changes_clear_pending(
-		vty, "/frr-isisd:isis/instance[area-tag='%s'][vrf='%s']", tag,
-		vrf_name);
+	snprintf(xpath, XPATH_MAXLEN,
```

Comment:
Can we use `snprintfrr` everywhere? It intentionally does not support `%n` which we don't use anywhere anyway…